packages/forklift-console-plugin/src/modules/Plans/views/create/CreatePlanWizard.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { FC, useState } from 'react';
+import { FormProvider, useForm } from 'react-hook-form';
+import { useForkliftTranslation } from 'src/utils';
+
+import { Form, Title, Wizard, WizardStep, WizardStepType } from '@patternfly/react-core';
+
+import { CreatePlanWizardFooter } from './components/CreatePlanWizardFooter';
+import { GeneralInformationForm } from './steps/GeneralInformationForm';
+import {
+  defaultCurrentStep,
+  getPlanWizardStepNames,
+  planStepIndexes,
+  PlanWizardStepId,
+} from './constants';
+
+export const CreatePlanWizard: FC = () => {
+  const { t } = useForkliftTranslation();
+  const form = useForm({ mode: 'onChange' });
+  const [currentStep, setCurrentStep] = useState<WizardStepType>(defaultCurrentStep);
+  const { formState, getValues } = form;
+  const formValues = getValues();
+  const stepNames = getPlanWizardStepNames(t);
```

Comment:
let introduce something like here:
https://github.com/kubevirt-ui/kubevirt-plugin/blob/main/src/utils/hooks/useKubevirtTranslation.ts#L15
so we dont need to send t to util files